core/src/avm2/globals/xml.rs

```diff
@@ -830,6 +830,42 @@ pub fn set_children<'gc>(
     Ok(xml.into())
 }
 
+// ECMA-357 13.4.4.34 XML.prototype.setLocalName ( name )
+pub fn set_local_name<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let xml = this.as_xml_object().unwrap();
```

Comment:
I suppose this `unwrap` here can never fail, right?

Reply:
Since the method is defined on the XML object I don't think it can fail, but correct me if I am wrong. And we have other XML methods that use `unwrap` in the same way.

Reply:
No, that was my intuition as well.